openapi-cli/src/main/java/io/ballerina/openapi/cmd/BallerinaCodeGenerator.java

```diff
@@ -160,7 +162,7 @@ public void generateClientAndService(String definitionPath, String serviceName,
                 openAPIDef, nullable, preGeneratedTypeDefNodes);
 
         SyntaxTree schemaSyntaxTree = ballerinaSchemaGenerator.generateSyntaxTree();
-        String schemaContent = Formatter.format(schemaSyntaxTree).toSourceCode();
+        String schemaContent = Formatter.format(schemaSyntaxTree).toString();
```

Comment:
@aneeshafedo shouldn't we have `toSourceCode()` in here?

Reply:
Yes we have to. It has been missed.